test/integration/options/debug.spec.js

```diff
@@ -25,8 +32,116 @@ describe('--debug', function() {
           );
           done();
         },
-        {stdio: 'pipe'}
+        'pipe'
+      );
+    });
+
+    it('should invoke --inspect-brk', function(done) {
+      var proc = invokeMocha(
+        ['--debug-brk', DEFAULT_FIXTURE],
+        function(err, res) {
+          if (err) {
+            return done(err);
+          }
+          expect(res, 'to have passed').and(
+            'to contain output',
+            /Debugger listening/i
+          );
+          done();
+        },
+        'pipe'
+      );
+
+      // debugger must be manually killed
+      setTimeout(function() {
+        proc.kill('SIGINT');
+      }, 1000);
+    });
```

Comment:
change to:
```
setTimeout(function() {
  process.kill(proc.pid, 'SIGINT');
}, 2000);
```
This way the `err.code` is 1 (not undefined which crashes `to have failed`).
A timeout of 1000 is too short, `res` may not be ready, yet => `to contain output` fails.

On my windows platform the four 'Node.js v8+' tests do pass.